Data/Sys/GameSettings/SOM.ini

```diff
@@ -12,11 +12,5 @@
 [ActionReplay]
 # Add action replay cheats here.
 
-[Video]
-
-[Video_Settings]
-
 [Video_Hacks]
-ImmediateXFBEnable = False
```

Comment:
Based on the wiki, the game probably still needs this.

Reply:
Yeah, I just saw it. Will change it back.